src/LanguageServer/Protocol/DefaultCapabilitiesProvider.cs

```diff
@@ -129,6 +129,7 @@ private static VSInternalServerCapabilities GetVSServerCapabilities()
             {
                 ProjectContextProvider = true,
                 BreakableRangeProvider = true,
+                DataTipRangeProvider = true,
```

Comment:
if this is expected to be active all the time in VS, it needs to be added here - https://github.com/dotnet/roslyn/blob/main/src/EditorFeatures/Core/LanguageServer/AlwaysActivateInProcLanguageClient.cs#L51

these capabilities only apply when we're in full LSP mode (feature flag, vscode).

Reply:
Moved.